android-setup.ps1

```diff
@@ -0,0 +1,41 @@
+$androidSdkHome = $ENV:ANDROID_HOME
+
+if ([string]::IsNullOrWhiteSpace($androidSdkHome))
+{
+    if ($ENV:OS -eq "Windows_NT")
+    {
+        $androidSdkHome = "C:\Program Files (x86)\Android\android-sdk\"
+    }
+    else
+    {
+        $androidSdkHome = Resolve-Path "~/Library/Developer/Xamarin/android-sdk-macosx"
+    }
+
```

Comment:
Should we check the $androidSdkHome path exists?
do we also support Linux as a build machine? Is it the same path as on macOS?